packages/mock/src/msw/index.ts

```diff
@@ -90,10 +93,18 @@ const generateDefinition = (
   const getResponseMockFunctionName = `${getResponseMockFunctionNameBase}${pascal(name)}`;
   const handlerName = `${handlerNameBase}${pascal(name)}`;
 
-  const mockImplementation = isReturnHttpResponse
-    ? `export const ${getResponseMockFunctionName} = (${isResponseOverridable ? `overrideResponse: Partial< ${returnType} > = {}` : ''})${mockData ? '' : `: ${returnType}`} => (${value})\n\n`
+  const addedSplitMockImplementations = splitMockImplementations.slice(
+    oldSplitMockImplementations.length,
+  );
+  splitMockImplementations.push(...addedSplitMockImplementations);
+  const mockImplementations = addedSplitMockImplementations.length
+    ? `//#region ${getResponseMockFunctionName} sub functions\n${addedSplitMockImplementations.join('\n\n')}\n//#endregion\n\n`
```

Comment:
If you want to include annotate, I think another way to express them using variable names would be appropriate. Or i think you can just delete it if it's not necessary.